lib/posix/posix_internal.h

```diff
@@ -42,6 +42,9 @@ struct posix_thread {
 
 	/* Queue ID (internal-only) */
 	uint8_t qid;
+
```

Comment:
This can probably be removed until e.g. `pthread_kill()` and `sigsetmask()` are implemented.